webdriver/search_test.go

```diff
@@ -67,9 +66,6 @@ func testSearch(t *testing.T, wd selenium.WebDriver, app AppServer, path string,
 	assert.Equal(t, "2dcontext/", text)
 }
 
-// NOTE(lukebjerring): Firefox, annoyingly, throws a TypeError querying shadowRoot because of a
```

Comment:
IMHO this shouldn't be moved. It belongs here. Perhaps adding something about how it refers to both `case "firefox":` blocks (i.e., the blocks in both functions below) would be appropriate to clarify why it's here.